docs/dev_sliding_dot_product.ipynb

```diff
@@ -0,0 +1,981 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #38.        _list7_ifft_v1(n, x, y)
</pre><p> </p><p>Need to create <code>y</code> in the line before line#38.</p><pre class="ql-syntax" spellcheck="false">y&nbsp;= np.empty(n, dtype=complex)   
</pre><p> </p><p>Check the other versions and fix them as well if needed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/939/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>